usmqe_tests/api/grafana/test_host_dashboard.py

```diff
@@ -0,0 +1,205 @@
+
+    .. test_result:: 2
+
+        Defined structure and structure from Grafana API are equivalent.
+    """
+    structure_defined = {
+        'Network': [
+            'Throughput',
+            'Dropped Packets Per Second',
+            'Errors Per Second'],
+        'At-a-Glance': [
+            'Health',
+            'Bricks',
+            'Brick Status',
+            'Memory Available',
+            'Memory Utilization',
+            'Swap Free',
+            'Swap Utilization',
+            'CPU Utilization',
+            'Brick IOPS'],
+        'Capacity & Disk Load': [
+            'Total Brick Capacity Utilization',
+            'Total Brick Capacity Utilization',
+            'Total Brick Capacity Available',
+            'Weekly Growth Rate',
+            'Weeks Remaining',
+            'Top Bricks by Capacity Percent Utilized',
+            'Top Bricks by Total Capacity',
+            'Top Bricks by Capacity Utilized',
+            'Disk Throughput',
+            'Disk IOPS',
+            'Disk IO Latency']}
+    structure = {}
+    for row in layout["dashboard"]["rows"]:
+        structure[row["title"]] = []
+        for panel in row["panels"]:
+            if panel["title"]:
+                structure[row["title"]].append(panel["title"])
+            elif "displayName" in panel.keys() and panel["displayName"]:
+                structure[row["title"]].append(panel["displayName"])
+
+    LOGGER.debug("defined layout structure = {}".format(structure_defined))
+    LOGGER.debug("layout structure in grafana = {}".format(structure))
+    pytest.check(
+        structure_defined == structure,
+        "defined structure of panels should equal to structure in grafana")
+
+
+def test_cpu_utilization(measured_cpu_utilization, cluster_reuse):
+    """@pylatest grafana/cpu_utilization
+    API-grafana: cpu_utilization
+    *******************
+
+    .. test_metadata:: author fbalak@redhat.com
+
+    Description
+    ===========
+
+    Check that Grafana panel *CPU Utilization* is showing correct values.
+    """
+    if cluster_reuse["short_name"]:
+        cluster_identifier = cluster_reuse["short_name"]
+    else:
+        cluster_identifier = cluster_reuse["integration_id"]
+    print(measured_cpu_utilization)
+    grafana = grafanaapi.GrafanaApi()
+    graphite = graphiteapi.GraphiteApi()
+    """@pylatest grafana/hosts
+    .. test_step:: 1
+        Send **GET** request to:
+        ``GRAFANA/dashboards/db/host-dashboard``.
+    .. test_result:: 1
+        JSON structure containing data related to layout is returned.
+    """
+
+    layout = grafana.get_dashboard("host-dashboard")
+    assert len(layout) > 0
+    dashboard_rows = layout["dashboard"]["rows"]
+
+    # get CPU Utilization panel from first row
+    panels = dashboard_rows[0]["panels"]
+    panel = [
+        panel for panel in panels
+        if "title" in panel and
+        panel["title"] == "CPU Utilization"]
```

Comment:
I guess that you expect only single such panel here, could you add a check for that here? Something like `pytest.check(len(panel) == 0)`. Also, as you can see, I'm not sure if using name `panel` for list of cpu panels fits, it would be better to do something like:

```
cpu_panels = ...
pytest.check(len(cpu_panels) == 0)
panel = cpu_panels[0]
...
```